tensorboard/tools/license_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+files=$(grep -rL "Copyright 20[0-9][0-9] The TensorFlow" \
+    --include=*.{cc,h,mm,java,sh,proto,bzl,py,html,ts,js} \
```

Comment:
It seems better to me I think to use a list of excluded extensions, so that if new ones are introduced, we default to requiring a license unless they definitely don't need one.  E.g. if we added a `.c` or a `.go` file this script wouldn't check for a license on it.

I tested this and I think if we use `--include=*.* --exclude=*.{pyc,json,png,wav,pbtxt,md,in,rst,cfg}` then it's the same result against the files we have checked in today.